draft-ietf-quic-transport.md

```diff
@@ -4740,6 +4740,14 @@ preferred_address (0x0d):
   be at least one unused active connection ID when the client initiates
   migration to the preferred address.
 
+: The Connection ID and Stateless Reset Token fields of a preferred address are
+  identical in syntax and semantics to the corresponding fields of a
+  NEW_CONNECTION_ID frame ({{frame-new-connection-id}}).  A server that chooses
+  a zero-length connection ID MUST NOT provide a preferred address.  Similarly,
```

Comment:
I don't think this is right. NEW_CONNECTION_ID frames don't ban zero-length CIDs. What NEW_CONNECTION_ID bans is switching between zero-length CIDs and non-zero-length CIDs. If the server is using a zero-length CID, it should be allowed to switch to a preferred address, and also use zero-length CIDs there.

Reply:
Thanks for taking this to the issue.

NEW_CONNECTION_ID can't [change to or from zero-length](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#section-19.15-6), so this is making this feature consistent with that.